docs/source/faq/faq.rst

```diff
@@ -70,7 +71,30 @@ or::
 
     >>> rm = ResourceManager('Path to library')
 
-or creating a configuration file as described in :ref:`intro-configuring`.
+If you get this error while trying to create a ``ResourceManager`` in Python built
+for Cygwin (see `https://www.cygwin.com/`) on Windows:
+
+  1. Check you are running the Cygwin build of Python by running ``python -VV``. If not follow
+     the troubleshooting steps for Windows::
+
+        $ python -VV
+        Python 3.9.10 (main, Jan 20 2022, 21:37:52)
+        [GCC 11.2.0]
+
+  2. Specify the location of the ``visa32.dll`` or ``visa64.dll`` using the ``linux`` syntax
+     and Cygwin paths by creating a `.pyvisarc` (:ref:`intro-configuring`) file::
+
+        $ cat ~/.pyvisarc
+        [Paths]
+        VISA library: /cygdrive/c/Windows/System32/visa64.dll
+
+     or::
+
+        rm = visa.ResourceManager('/cygdrive/c/Windows/System32/visa64.dll')
+
+     or::
+
+        rm = visa.ResourceManager('/cygdrive/c/Windows/System32/visa32.dll')
```

Comment:
It may be better to have a ``.. note::`` environment to mark it as a special case.

Reply:
Done! Your project is awesome, very glad to add my tiny contribution.

(I use Cygwin to avoid having to learn the quirks of Python on Windows, vs the quirks of Python on Linux that I know well. Python does do a good job of trying to be portable but some of the underlying OS mechanisms are just too different. I guess I could add a PR to detect Cygwin and look for the visa64.dll, however just creating a .pyvisa works well enough...)